alacritty/src/event.rs

```diff
@@ -1048,6 +1077,14 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                     self.ctx.display.cursor_hidden ^= true;
                     *self.ctx.dirty = true;
                 },
+                EventType::BlinkCursorTimeout => {
+                    // Disable blinking after timeout reached.
+                    let timer_id = TimerId::new(Topic::BlinkCursor, self.ctx.display.window.id());
+                    self.ctx.scheduler.unschedule(timer_id);
+                    self.ctx.display.cursor_hidden = false;
+                    *self.ctx.blink_cursor_timeout = true;
```

Comment:
I think `cursor_blink_timed_out` might be a better variable name? Or something like that at least. `blink_cursor_timeout` as a flag name doesn't sit quite right with me.